### ISSUE TITLE
error https://pdwebparty.herokuapp.com/

### ISSUE THREAD

**Issue by BorisMolch** [cuinjune/PdWebParty#8]
<img width="1131" alt="image" src="https://user-images.githubusercontent.com/33655927/208611404-e71e8b9f-a81e-411c-a646-e770a1f09961.png">

**Comment by tob-har** [comment]
heroku discarted their free plans, so I assume, thats why it is no available any more under that link.
you can still just host PdWebParty on your own, for example on glitch.me.

**Comment by cuinjune** [comment]
Yeah, I guess that's what happened. Thanks @tob-har, I will try to update this soon.

**Comment by tob-har** [comment]
hosted it on glitch here: https://pdwebparty.glitch.me/ works well, loading is a bit slow :)

**Comment by cuinjune** [comment]
Thank you, but it doesn't seem to be loading for me. I will try again soon. :)
